amg.c

```diff
@@ -624,6 +709,9 @@ main( hypre_int argc,
       HYPRE_ParCSRGMRESDestroy(pcg_solver);
       FOM2 = cum_nnz_AP * (HYPRE_Real)num_iterations / wall_time;
```

Comment:
Should this be included in the calculate-FOM region?

Reply:
good question, it does seem like it should be. Something similar is happening on line 671 though. I wonder if it is just calculating the time, as opposed to an actual value. Sorry I didn't add these annotation locations so I'm not quite sure, maybe @daboehme can confirm?